### PR TITLE
docs: 11.3 section of documentation updated according to new MFE

### PR DESCRIPTION
**Pull request by mehaknasir**
changes added as mentioned in this doc:
https://docs.google.com/document/d/1LTwB0F6oo-TlRUmIvWwT0-1dX4xDV_9j7j89YfOzJGM/edit#

ticket for the related changes:  https://2u-internal.atlassian.net/browse/INF-572

### Date Needed (optional)

features already released 

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert:
- [ ] Subject matter expert:
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [x] Product review:
- [ ] Partner support:
- [x] PM review:

FYI: Tag anyone else who might be interested in this PR here. @ayub02 
### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

[live link](https://draftinf-572.readthedocs.io/en/latest/sfd_discussions/add_post.html)

### Post-review

- [x] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst [hunk 1]
```diff
@@ -151,24 +173,42 @@ The following steps apply only to content-specific discussions.
...
+    The maximum size for an uploaded file is 1 MB.
+
+
+.. _Posting Anonymously:
+
+**********************
+Posting Anonymously
+**********************
+
+If the course staff has enabled the option to allow anonymous posts, a Post anonymously to peers checkbox
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/add_post.rst]: Bold font style for "Post anonymously to peers"

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst [hunk 2]
```diff
@@ -151,24 +173,42 @@ The following steps apply only to content-specific discussions.
...
 
-To participate in an ongoing discussion, you reply to the initial post by
-adding a response, or expand on a particular response by adding a comment.
 
-The same options for formatting the text and for adding links or images are
-available for responses and comments as for posts.
+.. note:: If you include an image with your post, include a description so that learners who use
+    screen readers to access the course can understand the image’s content and purpose. The description
+    also displays in place of the image if problems occur with the image file. If the image has no
+    functional purpose, leave the Description field empty and select This image is for decorative purposes
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/add_post.rst]: Change text to: "leave Image Description field empty"
Set font style bold for "Image Description"
Set font style bold for "This image is for decorative purposes only and does not require a description"

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst [hunk 3]
```diff
@@ -151,24 +173,42 @@ The following steps apply only to content-specific discussions.
...
+=====================================
+Add Image to a Post in a Course Unit
+=====================================
 
-==============================
-Add a Response or Comment
-==============================
+You can add an image while creating a post, click the Insert image button visible on top of the
+text editor to open a dialog box. Use this dialog box to upload and resize the image and add
+description and source of the image.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/add_post.rst:189]: Image is missing

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst [hunk 4]
```diff
@@ -226,22 +268,22 @@ edit or delete contributions from other learners.
...
    of the course or on the **Discussion** page.
 
-#. In the upper corner of the contribution, select the "More" icon (...).
-   A menu opens.
+#. In the top-right corner of the contribution, open the actions menu by clicking on the three dots (...)
 
-   .. image:: ../../../shared/students/Images/Disc_EditDelete.png
-    :width: 500
+.. image:: ../../../shared/students/Images/post_actions_menu.png
+    :width: 700
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/add_post.rst]: reduce width to 500

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst [hunk 5]
```diff
@@ -51,9 +51,16 @@ course and need an answer from the course team, be sure to create your post as
 a question, so that the course team sees that a response is required and
 responds appropriately.
 
+.. image:: ../../../shared/students/Images/discussion_or_question.png
+    :width: 700
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/add_post.rst]: reduce width to 500

**Review by ayub02 — CHANGES_REQUESTED** [review]
Center all images please.

File: en_us/open_edx_students/source/sfd_discussions/add_post.rst (hunk 3 above)
Comment (mehaknasir) [reply en_us/open_edx_students/source/sfd_discussions/add_post.rst:189]: https://draftinf-572.readthedocs.io/en/latest/sfd_discussions/add_post.html#add-image-to-a-post-on-the-discussion-page

Screen Shot 2022-10-05 at 4 22 15 PM

I can see the image here

**Review by ayub02 — APPROVED** [review]

**Pull request merged**